src/py42/response.py

```diff
@@ -22,6 +22,9 @@ def __init__(self, requests_response):
     def __getitem__(self, key):
         return self._data_root[key]
 
+    def __setitem__(self, key, value):
+        self._data_root[key] = value
+
     def __iter__(self):
         return self._iter
```

Comment:
I know this isn't part of this change, but I just noticed this. Since `self._iter` is a single generator result of `iter(self._data_root)`, you can only iterate over a response once. We should have `__iter__` return a new instance of `iter(self._data_root)` each time it's called, no?

Reply:
it's debatable whether or not being able to do this multiple times would be expected behavior, but one of the main goals of this project is approachability / ease of use, so I think I agree with you.